docs/runtime-function-reference.md

```diff
@@ -0,0 +1,691 @@
+| sort | string | The sort order for records in the leaderboard; possible values are "asc" or "desc". |
+| reset | string | The cron format used to define the reset schedule for the leaderboard. This controls when a leaderboard is reset and can be used to power daily/weekly/monthly leaderboards. |
+| metadata | table | The metadata you want associated to the leaderboard. Some good examples are weather conditions for a racing game. |
+| authoritative | bool | Mark the leaderboard as authoritative which ensures updates can only be made via the Lua runtime. No client can submit a score directly. |
+
+_Example_
+
+```lua
+local metadata = {
+  weather_conditions = "rain"
+}
+local id = "4ec4f126-3f9d-11e7-84ef-b7c182b36521"
+nk.leaderboard_create(id, "desc", "0 0 * * 1", metadata, false)
+```
+
+### logger
+
+__logger_error (message)__
+
+Write an ERROR level message to the server logs.
+
+_Parameters_
+
+| Param | Type | Description |
+| ----- | ---- | ----------- |
+| message | string | The message to write to server logs with ERROR level severity. |
+
+_Returns_
+
+(string) - The message which was written to the logs.
+
+_Example_
+
+```lua
+local message = ("%q - %q"):format("hello", "world")
+nk.logger_error(message)
+```
+
+---
+
+__logger_info (message)__
+
+Write an INFO level message to the server logs.
+
+_Parameters_
+
+| Param | Type | Description |
+| ----- | ---- | ----------- |
+| message | string | The message to write to server logs with INFO level severity. |
+
+_Returns_
+
+(string) - The message which was written to the logs.
+
+_Example_
+
+```lua
+local message = ("%q - %q"):format("hello", "world")
+nk.logger_info(message)
+```
+
+---
+
+__logger_warn (message)__
+
+Write an WARN level message to the server logs.
+
+_Parameters_
+
+| Param | Type | Description |
+| ----- | ---- | ----------- |
+| message | string | The message to write to server logs with WARN level severity. |
+
+_Returns_
+
+(string) - The message which was written to the logs.
+
+_Example_
+
+```lua
+local message = ("%q - %q"):format("hello", "world")
+nk.logger_warn(message)
+```
+
+### notifications
+
+__notifications_send_id (new_notifications)__
+
+Send one or more in-app notifications to a user. Have a look at the section on [in-app notifications](in-app-notifications.md).
+
+_Parameters_
+
+| Param | Type | Description |
+| ----- | ---- | ----------- |
+| new_notifications | table | The Lua table array of notifications to send. |
+
+_Example_
+
+```lua
+local subject = "You've unlocked level 100!"
+local content = nk.json_encode({
+  reward_coins = 1000
+})
+local user_id = "4c2ae592-b2a7-445e-98ec-697694478b1c" -- who to send
+local code = 101
+
+local new_notifications = {
+  { Subject = subject, Content = content, UserId = user_id, Code = code, Persistent = true}
+}
+nk.notifications_send_id(new_notifications)
+```
+
+
+
+### register hooks
+
+__register_after (func, msgname)__
```

Comment:
Shall we point them to this file for available message names?

https://github.com/heroiclabs/nakama/blob/master/server/runtime_messages.go

And then also mention that the msgname is case-insensitive